src/docs/admin/references/NotificationServiceReference.adoc

```diff
@@ -0,0 +1,187 @@
+==== Housekeeping
+
+Notifications in the notification service have a global life span defined in the
+application.properties located in * `PROACTIVE_HOME/dist/war/notification-service/WEB-INF/classes/application.properties`
+Passed that time, they will be removed from the database.
+
+Can be defined:
+
+- The call frequency of the housekeeping defined by a CRON expression:
+----
+# This cron expression determines the housekeeping call frequency.
+# Default value is 10 minutes: this will invoke the housekeeping mechanism
+# to remove every notifications which are set to be removed and has their scheduled time for removal reached.
+notifications.housekeeping.cronexpression=*/10 * * * *
+----
+
+- The life span of the notifications in seconds:
+----
+# Automatic remove notification delay (in seconds).
+# The time which notifications will be stored in the database
+# Set this time to 0 if you don't want the notifications to be removed.
+# Default value is 604800 seconds = 1 week
+notifications.housekeeping.removedelay=604800
+----
+==== Notification methods
+
+===== Email
+
+Email notification method uses SMTP protocol to send emails, thus the notification
+service needs to have knowledge and connect to a SMTP server to use it.
+Relevant data is specified in the properties file located
+* `PROACTIVE_HOME/dist/war/notification-service/WEB-INF/classes/email.properties`
+
+By default Transport Layer Security is active, change to false to deactivate it:
+----
+mail.smtp.starttls.enable=true
+----
+So is the authentication to the STMP server.
+----
+mail.smtp.auth=true
+----
+
+Provide the server port used to connect
+----
+mail.smtp.port=
+----
+
+The credentials used to connect
+----
+mail.smtp.username=
+mail.smtp.password=
+----
+
+The host of the SMTP server
+----
+mail.smtp.host=
+----
+
+The email used to broadcast the emails is defined in the application.properties file,
+go to <<Housekeeping>> to see where it is located.
+It is the email that will appear in the "From" field of the sent emails
+----
+notification.email.from=
+----
+
+[[_notification_third_party]]
+===== Third party
+
+====== Configuration
+
+The Third party notification method executes a script when an event occurs.
+Location of the script is specified either via a path, relative to the Scheduler or absolute.
+
+----
+notification.script.path=dist/war/notification-service/WEB-INF/classes/exampleScript.sh
+----
+
+Or a URL
+
+----
+notification.script.url=
+----
+
+Note that to the URL will be processed if the path is empty
+
+An optional parameter can be provided to specify the engine to be used to execute the script
+if not provided the extension of the file will determine the appropriate engine to use
+
+----
+#optional parameter for specifying Script Engine
+notification.script.engine=bash
+----
+
+Finally multiple script execution can be enable to execute the script for each user
+that has an active subscription for an event and has Third party notification method activated.
```

Comment:
Parallel script executions can be activated to support multiple users in parallel. Users must have an active subscription for an event and a Third party notification method activated.

Reply:
It is actually sequentially executed for each user